### ISSUE TITLE
Ember checkbox click event not firing

### ISSUE THREAD

**Issue by dgavey** [emberjs/ember.js#10146]
We have an internal modal system we made with Ember. A background div is placed over the whole screen and an inner modal window is placed on top of it.  I recently added a click event to the background div that fades out the screen so it would close the modal when clicked. Unfortunately it seems that the method I choose to do this with seems to be causing some errors with Ember.

Specifically checkboxes don't consistently fire to toggle their value.  Jquery click events (Like those used to start a file upload with a file input box) are also not firing.

Here is a JSBIN with the checkbox problem structured similar to how our modals work.

http://jsbin.com/siboyekiha/5/

**Comment by dgavey** [comment]
Never reported a bug to the Ember community before, a bit surprised it hasn't got a reply yet.   I'm hoping I didn't report it incorrectly.

**Comment by OrKoN** [comment]
Hi,

Looks like it's a problem in Ember.js but I am not an expert to claim anything. See my experiments below - maybe, it'd help:

http://jsbin.com/wobuvisoqe/2/edit?html,js,output

I've assigned a handler to the innerWindow using jquery and I do stop propagation in there (e.g. I've removed the action on the innerWindow). Now the checkbox works for some reason but the button does not. I think it's because if the stopPropagation is called, normal actions are not propagated to the main Ember's handler which is on the body element (again, that's only my guess).
